### PR TITLE
Add FeatureLayers and GeoJSON layers to leaflet.d.ts

### PR DESCRIPTION
**Pull request by sigh71**
Added some missing types and properties, and fixed a typo.

### CODE REVIEW

File: leaflet/leaflet.d.ts [hunk 1]
```diff
@@ -256,11 +259,43 @@ declare namespace L {
...
+        bringToBack(): this;
+        getBounds(): LatLngBounds;
+    }
+
+    export interface GeoJSONOptions {
+        pointToLayer: (feature: any, latLng: LatLng) => void;
+        style: (feature: any) => void;
+        onEachFeature: (feature: any, layer: Layer) => void;
+        filter: (feature: any) => boolean;
+        coordsToLatLng: (coords: number[]) => LatLng;
```
Comment (alejo90) [inline leaflet/leaflet.d.ts]: The parameter's type can be more specific: `[number, number] | [number, number, number]`

File: leaflet/leaflet.d.ts [hunk 2]
```diff
@@ -599,7 +635,9 @@ declare namespace L {
         opacity?: number;
     }
 
-    export interface Tooltip extends Layer {}
+    export interface Tooltip extends Layer {
+        options: TooltipOptions;
```
Comment (alejo90) [inline leaflet/leaflet.d.ts]: I couldn't find this on the docs.

File: leaflet/leaflet.d.ts [hunk 3]
```diff
@@ -847,6 +885,7 @@ declare namespace L {
         scrollWheelZoom: Handler;
         tap: Handler;
         touchZoom: Handler;
+        options: MapOptions;
```
Comment (alejo90) [inline leaflet/leaflet.d.ts]: Couldn't find this one on the docs either.

**Review by alejo90 — CHANGES_REQUESTED** [review]
First of all, thanks for the help. There are a couple of minor issues but other than that, it looks great. Also, make sure your next commit passes the CI tests.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**